cmd/osbuild-worker/main.go

```diff
@@ -162,9 +166,29 @@ func main() {
 
 	_, err := toml.DecodeFile(configFile, &config)
 	if err == nil {
+		logrus.SetOutput(os.Stdout)
+		if config.Logrus != nil {
+			logLevel, err := logrus.ParseLevel(config.Logrus.LogLevel)
+			logrus.SetReportCaller(true)
```

Comment:
You can skip this, it is too noisy sometimes.

Reply:
I agree, I'm gonna drop it I think.